set.mm

```diff
@@ -20546,6 +20552,17 @@ satisfied directly but only indirectly (outside of Metamath) by the
      with the same variable.  In this case, the first antecedent becomes
      ` -. x = x ` and the axiom still holds.
 
+     This axiom is mostly used to eliminate conditions requiring set variables
+     be distinct (cf. ~ ax6ev and ~ ax6e , for example) in proofs.  In
+     practice, theorems beyond elementary set theory do not really benefit from
+     such eliminations, so direct or indirect application of this axiom is
+     restricted now.  You need to explicitly confirm its use in case you see a
```

Comment:
Rather than "restricted", maybe "discouraged" ?
"restricted" sounds a bit like it would be impossible.

Reply:
done